backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AutoMapper;
+using Squirrel.Core.Common.DTO.Projects;
+using Squirrel.Core.BLL.Interfaces;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : IProjectService
+    {
+        // private readonly SquirrelCoreContext _context;
+        private readonly IMapper _mapper;
+
+        public ProjectService(/* SquirrelCoreContext context, */ IMapper mapper)
```

Comment:
In this service you will use context and mapper so you can just inhearite base service

Reply:
Fixed